src/modules/src/controller_lee_payload.c

```diff
@@ -249,11 +289,15 @@ static void runQP(const struct QPInput *input, struct QPOutput* output)
     struct vec n5 = computePlaneNormal(statePos3, statePos, plStPos,  radius, l3, l1);
     struct vec n6 = computePlaneNormal(statePos3, statePos2, plStPos, radius, l3, l2);
     // printf("%f\n",(double)workspace->data->A->nzmax);
-    c_float Ax_new[27] = {1, n1.x, n2.x, 1, n1.y, n2.y, 1, n1.z, n2.z, 1, n3.x, n4.x, 1, n3.y, n4.y, 1, n3.z, n4.z, 1, n5.x, n6.x, 1, n5.y, n6.y, 1, n5.z, n6.z, };
-    // c_int Ax_new_idx[27] = {0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15, 16, 17, 18, 19, 20, 21, 22, 23, 24, 25, 26};
-    c_int Ax_new_n = 27;
-    c_float l_new[9] =  {F_d.x,	F_d.y,	F_d.z, -INFINITY, -INFINITY, -INFINITY, -INFINITY, -INFINITY, -INFINITY,};
-    c_float u_new[9] =  {F_d.x,	F_d.y,	F_d.z, 0, 0,  0, 0,  0, 0};
+    // c_float Ax_new[37] = {1, attPoint.x,  n1.x, n2.x, 1, n1.y, n2.y, 1, n1.z, n2.z, 1, n3.x, n4.x, 1, n3.y, n4.y, 1, n3.z, n4.z, 1, n5.x, n6.x, 1, n5.y, n6.y, 1, n5.z, n6.z, };
+    c_float Ax_new[45] = {
+      1, attPoint.z, -attPoint.y, n1.x, n2.x, 1, -attPoint.z, attPoint.x, n1.y, n2.y, 1, attPoint.y, attPoint.x, n1.z, n2.z,
+      1, attPoint2.z, -attPoint2.y, n3.x, n4.x, 1, -attPoint2.z, attPoint2.x, n3.y, n4.y, 1, attPoint2.y, attPoint2.x, n3.z, n4.z,
+      1, attPoint3.z, -attPoint3.y, n5.x, n6.x, 1, -attPoint3.z, attPoint3.x, n5.y, n6.y, 1, attPoint3.y, attPoint3.x, n5.z, n6.z,
+    };
+    c_int Ax_new_n = 45;
+    c_float l_new[12] =  {F_d.x,	F_d.y,	F_d.z,  M_d.x,  M_d.y,  M_d.z,  -INFINITY, -INFINITY, -INFINITY, -INFINITY, -INFINITY, -INFINITY,};
+    c_float u_new[12] =  {F_d.x,	F_d.y,	F_d.z,  M_d.x,  M_d.y,  M_d.z, 0, 0,  0, 0,  0, 0};
```

Comment:
This might require us to increase the task size even more: (45+12+12)*4 = 276 Bytes for these lines alone.

Reply:
I was just making sure that we can access all of the values and the  changes are correct, but we can update this later to change it in a loop as discussed before.